### PR TITLE
store the kubeadmin password when installing the endurance cluster

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

File: ci-operator/templates/openshift/endurance/endurance-install.yaml [hunk 1]
```diff
@@ -431,7 +431,8 @@ objects:
         
         # the ci-operator pod will copy this secret into the ci namespace to persist
         # it for future jobs to use when accessing the long lived cluster.
-        oc create secret generic endurance-cluster-credentials-${CLUSTER_TYPE}-new -n ${NAMESPACE} --from-file=kubeconfig=/tmp/artifacts/installer/auth/kubeconfig --from-file=metadata.json=/tmp/artifacts/installer/metadata.json
+        grep -o "password[^\"]*" /tmp/artifacts/installer/.openshift_install.log | awk '{print $2}'> /tmp/artifacts/installer/auth/kubeadmin
+        oc create secret generic endurance-cluster-credentials-${CLUSTER_TYPE}-new -n ${NAMESPACE} --from-file=kubeconfig=/tmp/artifacts/installer/auth/kubeconfig --from-file=metadata.json=/tmp/artifacts/installer/metadata.json --from-file=kubeconfig=/tmp/artifacts/installer/auth/kubeadmin
```
Comment (abhinavdahiya) [inline ci-operator/templates/openshift/endurance/endurance-install.yaml]: the kubeadmin password is already created by the installer in the <INSTALL_DIR>/auth/kubeadmin-password

https://github.com/openshift/installer/blob/013bcf84c2b67cf7150f54788edd102a0e3349c9/pkg/asset/password/password.go#L14

Comment (abhinavdahiya) [reply ci-operator/templates/openshift/endurance/endurance-install.yaml]: secondly, the the last from file as the same key as the first from file.

Comment (bparees) [reply ci-operator/templates/openshift/endurance/endurance-install.yaml]: thanks @abhinavdahiya (and good catch on the key..lazy cutting+pasting bites again).

PR is updated.

File: ci-operator/templates/openshift/endurance/endurance-install.yaml [hunk 2]
```diff
@@ -431,7 +431,7 @@ objects:
         
         # the ci-operator pod will copy this secret into the ci namespace to persist
         # it for future jobs to use when accessing the long lived cluster.
-        oc create secret generic endurance-cluster-credentials-${CLUSTER_TYPE}-new -n ${NAMESPACE} --from-file=kubeconfig=/tmp/artifacts/installer/auth/kubeconfig --from-file=metadata.json=/tmp/artifacts/installer/metadata.json
+        oc create secret generic endurance-cluster-credentials-${CLUSTER_TYPE}-new -n ${NAMESPACE} --from-file=kubeconfig=/tmp/artifacts/installer/auth/kubeconfig --from-file=metadata.json=/tmp/artifacts/installer/metadata.json --from-file=kubeadmin=/tmp/artifacts/installer/auth/kubeadmin-password
```
Comment (smarterclayton) [inline ci-operator/templates/openshift/endurance/endurance-install.yaml]: Is this consistent with what the installer does?

Comment (smarterclayton) [reply ci-operator/templates/openshift/endurance/endurance-install.yaml]: I guess it doesn't matter.

**Pull request merged**